pack/fhv/fhvc.json

```
@@ -107,6 +107,331 @@
+    "cost": null,
+    "deck_limit": 0,
+    "encounter_code": "written_in_rock",
+    "encounter_position": 7,
+    "faction_code": "neutral",
+    "illustrator": "Carlos Palma Cruchaga",
+    "name": "Mine Cart",
+    "pack_code": "fhvc",
+    "position": 7,
+    "quantity": 1,
+    "subname": "Reliable but Broken",
+    "text": "<b>Forced</b> - At the end of the enemy phase: Move Mine Cart once. If it moves off the rail line, each investigator is defeated and suffers 1 physical trauma.\n[reaction] When Mine Cart moves, spend 1 switch: Either cancel the move, or move an additional time. (Group limit twice per phase.)",
+    "traits": "Vehicle. Cart.",
+    "type_code": "asset"
+  },
+  {
+    "back_flavor": "The lantern in the shed is lit. Who was here before you?",
+    "back_text": "As an additional cost to enter Control Station, investigators at an adjacent location must spend 2[per_investigator] clues, as a group.",
+    "clues": 2,
+    "code": "10508",
+    "double_sided": true,
+    "encounter_code": "written_in_rock",
+    "encounter_position": 8,
+    "faction_code": "mythos",
+    "flavor": "The back of the station is filled with crates of very old dynamite and stocks of gunpowder.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Control Station",
+    "pack_code": "fhvc",
+    "position": 8,
+    "quantity": 1,
+    "shroud": 1,
+    "text": "[fast] During your turn: Swap 2 adjacent [[Rail]] locations. (Group limit once per game.)",
+    "traits": "Rail. Station.",
+    "type_code": "location",
+  }, 
+  {
+    "back_flavor": "The inviting glow of lamps and the scent of fresh air surely means there is another exit.",
+    "back_text": "Rail Exit cannot be slid or swapped.",
+    "clues": 0,
+    "code": "10509",
+    "double_sided": true,
+    "encounter_code": "written_in_rock",
+    "encounter_position": 9,
+    "faction_code": "mythos",
+    "flavor": "Scratch marks score the walls and beams nearby. You shudder to think of the frenzied creatures who have excaped North Point Mine to hunt for their next meal.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Rail Exit",
+    "pack_code": "fhvc",
+    "position": 9,
+    "quantity": 1,
+    "shroud": 1,
+    "text": "Rail Exit cannot be slid or swapped.",
+    "traits": "Rail.",
+    "type_code": "location",
+    "victory": 1
+  }, 
+  {
+    "back_flavor": "These tracks really don't look safe.",
+    "back_name": "Rail Tunnel",
+    "clues": 1,
+    "code": "10510a",
+    "double_sided": true,
+    "encounter_code": "written_in_rock",
+    "encounter_position": 10,
+    "faction_code": "mythos",
+    "flavor": "The crystals emanate a strange light.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Left Turn",
+    "pack_code": "fhvc",
+    "position": 10,
+    "quantity": 1,
+    "shroud": 2,
+    "text": "[action] [action]: Swap 2 adjacent [[Rail]] locations. (Group limit once per game.)",
+    "traits": "Rail.",
+    "type_code": "location",
```

```suggestion
    "type_code": "location"
```
All cases like this need to be fixed